acra-censor/handlers/querycapture_handler.go

```diff
@@ -219,8 +220,15 @@ func AppendQueries(queries []*QueryInfo, openedFile *os.File) error {
 
 func SerializeQueries(queries []*QueryInfo) ([]byte, error) {
 	var linesToAppend []byte
+	var tempQueryInfo = &QueryInfo{}
 	for _, queryInfo := range queries {
-		jsonQueryInfo, err := json.Marshal(queryInfo)
+		queryWithHiddenValues, err := sqlparser.RedactSQLQuery(queryInfo.RawQuery)
```

Comment:
may we copy implementation and refactor it to accept placeholder as param? I think in a future we will want to change prefix to something else than "redactedN". Plus tests will not be so hardcoded as now (if I wouldn't know about exact value for placeholder then I wouldn't understand why we expect "redacted1" after function call).